video_worker/video_images.py

```diff
@@ -136,6 +137,13 @@ def upload(self, generated_images):
             logger.error(': Invalid Storage Bucket for Video Images')
             return
 
+        bucket_location = bucket.get_location()
```

Comment:
again, why we need this multiple call ways to get AWS bucket object?

Reply:
When the EC2 instance is located at different region from the S3 storage, it returns and can handling the bucket, but when user try to upload/download from the bucket, it causes connection reset. So, one need to get the bucket from the belonged region.

You can check this link
https://github.com/boto/boto/issues/2207#issuecomment-60682869

Thanks,

Reply:
Umm. EC2 connection... It not about getting connection to the ec2, it's about the "default region".
Problem is happens when the S3 bucket is located at different to the "default region". By default, boto connects to the default region, but we need to connect to the specific region(which the bucket belonged).

Cleaner way to get the connection to the bucket's region is specifying the region name. But to do that we need additional configuration parameter for each bucket. So I tried to find way to get the connection without configure the region parameter.

IMHO, less configuration is better if the behavior can be same..

Thank,